Tasks/VSMobileCenterUpload/vsmobilecenterupload.ts

```diff
@@ -57,7 +58,12 @@ function responseHandler(defer, err, res, body, handler: () => void) {
 
     tl.debug(`---- http call status code: ${res.statusCode}`);
     if (res.statusCode < 200 || res.statusCode >= 300) {
-        let message = JSON.stringify(body) || `http response code: ${res.statusCode}`;
+        let message = JSON.stringify(body);
+        if (!message) {
+            message = `http response code: ${res.statusCode}`;
+        } else {
+            message = message.concat(os.EOL + `http response code: ${res.statusCode}`);
+        }
         defer.reject(message);
```

Comment:
I'd also use ```${JSON.stringify(err)}``` instead of just ```${err}``` at the end of the file. Also, ```tl.error(...)``` call won't hurt before ```tl.setResult(...)```